my-app/src/App.tsx

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 import "./App.css";
+import TelaLogin from "./components/TelaLogin/screenlogin";
```

Comment:
para se adequar com a convenção
```suggestion
import TelaLogin from "./components/tela-login/screen-login";
```